### DISCUSSION TITLE
How to import capacitor plugins from "capacitor-4" branch instead of "main"?

### DISCUSSION THREAD

**Discussion by egeozer** [ionic-team/capacitor#5589] [General]
Hi, 

I'm trying to import to my project @capacitor/local-notifications from the "capacitor-4" branch since there was a recent fix in this particular branch that fixes the app crashing issue for android 12 phones. 

I'm currently on latest capacitor plugins, however I have no idea how to make it target "capacitor-4" branch instead of the main branch. Anyone knows how to do this?

Thanks a lot.

**Comment by thor9n** [comment]
I also wonder how to achieve this. I've managed to install the package as following:
```json
"dependencies": {
   "capacitor-plugins": "github:ionic-team/capacitor-plugins#pull/1436",
   ...
```

But when running `ionic cap sync` it doesn't interpret it as a capacitor plugin and skips it.
